default/buildings.txt

```diff
@@ -999,6 +1026,7 @@ BuildingType
             Planet environment = Adequate
         ]
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
a BLD_TERRAFORM removes itself on the turn it is constructed, so only causes a one turn hang in any other pending terraforms, which is not a reason to prohibit them.

As for BLD_COLONY_BASE, it would be the same issue -- it creates a ship and then destroys itself, and so need to block another one.  Especially since this particular piece of content is not currently used; if someone wants to reintroduce it they can decide how such issues fit with their vision of it.  So let's leave it alone for now.